swo/swoinfo/db.go

```diff
@@ -22,11 +22,11 @@ func DBInfo(ctx context.Context, conn *pgx.Conn) (*DB, error) {
 	if err != nil {
 		return nil, err
 	}
-	if info.ID == uuid.Nil {
+	if !info.ID.Valid {
 		return nil, fmt.Errorf("no database ID")
```

Comment:
```suggestion
		return nil, fmt.Errorf("invalid database ID")
```

Reply:
talked about this offline -- `Valid` in this context refers to the state of the `ID` (`NullUUID` type), `!Valid` means a `NULL` value was scanned and thus there is no database ID.